tools/scripts/seed-exams/README.md

```diff
@@ -0,0 +1,33 @@
+## WARNING: Never change any of the ID's or delete anything. Mark things as deprecated instead.
+
+### How to create a new exam:
+
+1. Add the exam file in the `exams` folder
+
+2. Add the exam meta data:
+
+- `_id`: This should match the `id` in the exam challenge markdown file,
+- `title`: This should match the `title` in the exam challenge markdown file,
```

Comment:
If this is a requirement, can we validate it before running the script?

Reply:
Are they [not being validated?](https://github.com/freeCodeCamp/freeCodeCamp/pull/50836/files#diff-394e55c5ba66a5c19c7116353096f881508892d6199dc8aa4fc2cabcf6383fbeR48) - If I don't have the required properties, it fails.